servers/v3-proxy-api/src/graph/types.ts

```diff
@@ -66,18 +66,39 @@ export type GetSearchResponseCompleteTotal = GetSearchResponseComplete & {
   total: string;
 };
 
-export type GetResponseComplete = {
-  //todo: add top level fields
-  //e.g status, complete - as they are not mapped by developer portal docs
-  // note that complete returns 1 for detailType=simple and
-  // detailType=complete when querying v3 API directly
-  // since: number;
-  // maxActions: number;
-  // error: ?;
-  // search_meta: {search_type: "normal"}
-  list: { [key: string]: ListItemObjectComplete };
+/**
+ * Represents a static response that we still need to return from Get with static
+ */
+export type GetStaticResponse = {
+  // Always return 30.
+  maxActions: number;
   cachetype: string;
 };
+
+/**
+ * Represents a static response that we still need to return from Get with empty values when a client requests `shares=1`
+ */
+export type GetSharesResponse = {
+  // empty data fields that used to have a use, but now are empty and Android will crash without.
+  recent_friends: [];
+  auto_complete_emails: [];
+  unconfirmed_shares: [];
+};
+
+export type GetTopLevelDefaultResponse = {
+  complete: number; // 0 if preg_match('/^[0-9]*$/', $since) && $since > 0, else 1
```

Comment:
aha, _that's_ why it didn't match `detailType=complete`

Reply:
yea i still gotta understand and come back to this one.